convert2rhel/main.py

```diff
@@ -115,6 +115,8 @@ def main_locked():
         process_phase = ConversionPhase.PRE_PONR_CHANGES
         pre_conversion_results = actions.run_actions()
 
+        loggerinst.info("1 - before rollback: %s" % len(backup.backup_control._restorables))
```

Comment:
Looks like forgotten debug print, also on line 157 and 333

Reply:
Thanks!